views/css/ps_socialfollow.css

```diff
@@ -0,0 +1,5 @@
+.discord {
```

Comment:
@marsaldev this class is too generic, let's go with the, maybe:

```suggestion
.ps-socialfollow-discord {
```

cc @NeOMakinG

Reply:
I thought about it 😅 but I have aligned the name to old classes.
But I agree with you 👍🏻

Reply:
Agree